cypress/e2e/data-stream.cy.js

```diff
@@ -0,0 +1,59 @@
+import generateUniqueName from '../support/utils'
+import selectors from '../support/selectors'
+
+let dataStreamName
+
+describe('Data Stream spec', () => {
+  beforeEach(() => {
+    dataStreamName = generateUniqueName('DataStream')
+
+    cy.login()
+    cy.openProductThroughSidebar('data-stream')
+  })
+
+  it('should create and delete a data stream with the standard connector', () => {
```

Comment:
O delete ficou no forEach, que é fora do teste.

Pode deixar apenas 'should create a data stream with the standard connector'